api/v1alpha1/stresschaos_types.go

```diff
@@ -225,13 +225,25 @@ func (in *Stressors) Normalize() (string, error) {
 					in.MemoryStressor.Size)
 			}
 		}
+
+		if in.MemoryStressor.Options != nil {
+			for _,v := range in.MemoryStressor.Options{
+				stressors += fmt.Sprintf(" %v ", v)
+			}
+		}
 	}
 	if in.CPUStressor != nil {
 		stressors += fmt.Sprintf(" --cpu %d", in.CPUStressor.Workers)
 		if in.CPUStressor.Load != nil {
 			stressors += fmt.Sprintf(" --cpu-load %d",
 				*in.CPUStressor.Load)
 		}
+
+		if in.CPUStressor.Options != nil {
+			for _,v := range in.CPUStressor.Options{
+				stressors += fmt.Sprintf(" %v ", v)
+			}
+		}
```

Comment:
What about if user set the `--cpu-load ` option? How would we merge the adding option and the existed option.

Reply:
@Yiaser  by stress-ng itself will take over the repeated options ( tested: later one will replace former one)

A test case is : （ --cpu is a more clear test case, Top to find there is only on  Processor of "stress-ng-cpu "）
`stress-ng --cpu-load 10 --cpu 2 --cpu  1`